cmd/zed/api/get/get.go

```diff
@@ -149,9 +144,9 @@ func parseExpr(spaceID api.SpaceID, expr string) (*api.SearchRequest, error) {
 		return nil, err
 	}
 	return &api.SearchRequest{
-		Space: spaceID,
-		Proc:  proc,
-		Dir:   -1,
+		Dir:  -1,
```

Comment:
I like the idea of keeping this to keep the app running while we bring up RunDAG and RunZed endpoints eventually deprecating SearchRequest.